oscar_odin/resources/catalogue.py

```diff
@@ -137,28 +135,3 @@ class ProductResource(OscarCatalogueResource):
     children: Optional[List["ProductResource"]] = odin.ListOf.delayed(
         lambda: ProductResource, null=True
     )
-
-    def clean(self):
```

Comment:
Please make sure that there is validation that ensures partner and upc (and currency?) are present when either price or availability is not None.

The mapping checks for these to be present and will fail silently if this is not true